### DISCUSSION TITLE
LVM mount is recommended for ozone data directory?

### DISCUSSION THREAD

**Discussion by cloudcheflabs** [apache/ozone#5013] [FAQ]
Hi,

I have mounted 2 disk as lvm with the path of `/export`.
and then, I have configured `hdds.datanode.dir` and `dfs.datanode.data.dir` to `/export/ozone/dn-data`.

Is this recommended way to configure data node directories using lvm mount?

- Kidong.

**Comment by kerneltime** [comment]
It should be ok, Ozone does not care about how the Linux volumes are configured. OM tends to be latency sensitive with high IOPS needs so it is best to use NVMe for OM metadata. What is the concern you have for LVM?

**Reply by cloudcheflabs** [reply]
@kerneltime Thank you for your reply. I just want to know what is best way to configure data node directories for multiple disks.
